tmt-app/src/Components/MajorPage.jsx

```diff
@@ -14,14 +14,7 @@ function MajorPage() {
   searchData(params.major);
   return (
     <Container sx={{ minWidth: "70%" }}>
-      <Breadcrumb
-        previousPage={{
-          category: "Major",
-          major: data.name,
-          parent: "Home",
-          path: "",
-        }}
-      />
+      <Breadcrumb previousPage={["Major", data.name, "Home", ""]} />
```

Comment:
as mentioned above and in our discussion, i think breadcrumb should take a prop `breadcrumbs` or `links` or anything to signify an array of `breadcrumb`s as a type. 

as mentioned in the other really long comment as well, if we're thinking about types, we can declare (at least in our minds) that the prop `breadcrumbs` can expect `Breadcrumb[]` where `Breadcrumb` (the type) follows the shape `{url?: string, label: string}` where url is optional and if present, renders a link, if not, just plain text.